src/mscorlib/model.xml

```diff
@@ -8216,6 +8437,20 @@
       <Member MemberType="Property" Name="FileName" />
       <Member Status="ImplRoot" Name="#ctor(System.String,System.String,System.Int32)" />
     </Type>
+    <Type Name="System.IO.FileStream">
```

Comment:
Why did you need to expose FileStream? I would like to avoid doing that if possible.